frame/election-provider-multi-phase/src/benchmarking.rs

```diff
@@ -382,6 +382,20 @@ frame_benchmarking::benchmarks! {
 		assert!(<MultiPhase<T>>::queued_solution().is_some());
 	}
 
+	force_rotate_round {
+
+	}: _(RawOrigin::Root, Phase::Off)
+	verify {
+		assert_eq!(<MultiPhase<T>>::round(), 2);
+	}
+
+	force_rotate_round_from_signed {
+		<CurrentPhase<T>>::put(Phase::Signed);
+	}: force_rotate_round(RawOrigin::Root, Phase::Signed)
+	verify {
+		assert_eq!(<MultiPhase<T>>::round(), 2);
+	}
+
```

Comment:
I'm happy to have these be manually weighed as well, given their trivial state transition.